src/main/java/com/exasol/validation/GitRepositoryValidator.java

```diff
@@ -104,8 +130,10 @@ protected void validateChanges(final ReleaseLetter changes, final String version
     private void validateVersionInChanges(final ReleaseLetter changes, final String version) {
         final Optional<String> versionNumber = changes.getVersionNumber();
         if ((versionNumber.isEmpty()) || !(versionNumber.get().equals(version))) {
-            throw new IllegalStateException("E-RR-VAL-6: " + changes.getFileName()
-                    + " file does not mention the current version. Please, follow the changes file's format rules.");
+            this.validationReport.addFailedValidations("E-RR-VAL-6", "The file '" + changes.getFileName()
+                    + "' does not mention the current version. Please, follow the changes file's format rules.");
```

Comment:
add current version here. By that users can directly see if RR has detected the wrong version somehow or simply copy it.

Reply:
You can see the version in the file name. The file's name is already validated at this stage, so the version is correct

Reply:
:+1: